.eslintrc

```diff
@@ -1,5 +1,8 @@
 {
   "parser": "@typescript-eslint/parser",
   "plugins": ["@typescript-eslint"],
-  "extends": ["plugin:@typescript-eslint/recommended", "prettier"]
+  "extends": ["plugin:@typescript-eslint/recommended", "prettier"],
+  "rules": {
+    "@typescript-eslint/ban-types": "off"
```

Comment:
Can you expand on why you need this?

Reply:
TypeScript misinterprets the types from AssemblyScript for `BigInt` and makes a wrong suggestion